app/config/routing.yml

```diff
@@ -1,6 +1,9 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
+fos_oauth_server_token:
```

Comment:
In main ApiBundle it doesn't work, because you need to be login to get under /api.